src/s1reader/s1_reader.py

```diff
@@ -860,9 +850,106 @@ def _burst_from_safe_dir(safe_dir_path: str, id_str: str, orbit_path: str, flag_
     else:
         msg = f'measurement directory NOT found in {safe_dir_path}'
         msg += ', continue with metadata only.'
-        print(msg)
+        # print(msg)
```

Comment:
Is it intentional to have a comment here? Don't you want to just print the message for debugging purposes?

Reply:
ah sorry, I had turned this off while downloading the cycle of metadata so it didn't print the message 3 * 10 * (number of frames in a cycle) times. I'll turn it back on, but it would be nice to have it be a `.debug` logging message once we switch in python logging or do #25